src/plugins/ui_core_views/cell_evaluation/evaluator.ts

```diff
@@ -456,7 +456,7 @@ export class Evaluator {
 
   private checkCollision(formulaPosition: CellPosition): (i: number, j: number) => void {
     const { sheetId, col, row } = formulaPosition;
-    const checkCollision = (i: number, j: number) => {
+    return (i: number, j: number) => {
```

Comment:
don't we loose the function name when inspecting the callstack ?

Reply:
That's true, I didn't think arrow function could be named in the callstack, even if they were assigned to a variable. I'll change it back.